Core/CoreTests/Database/Models/RubricTests.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import XCTest
+@testable import Core
+
+class RubricTests: CoreTestCase {
+
+    override func setUp() {
```

Comment:
Remove? Or is it intentionally blank so that the super setUp doesn't get called? Seems like that would break much of the `CoreTestCase` use.